src/commands/remember/recall.js

```diff
@@ -0,0 +1,39 @@
+const { ApplicationCommandOptionType, Attachment } = require('discord.js');
+const { getRememberedMessage } = require("../../utils/rememberMessages");
+const { ephemeral } = require('../../../config.json');
+var fs = require('fs');
+//remembers a message based on a message id parameter
+module.exports = {
+    name: 'recall',
+    description: 'creates a JSON of all the saved message',
+    //logic, 
+    callback: async (client, interaction) => {
+        try {
+            //get the json
+            //? There is probably a way to do this so we don't have to have a fle created to save to
+            //? Will probably be resolved once DB is created
+            await interaction.deferReply({ ephemeral: false })
+            const jsonFilePath = './src/rememberedMessages.json';
+            const messageObj = []
+            const messages = getRememberedMessage();
+            messages.forEach(m => messageObj.push(m))
+            interaction.editReply("Success")
```

Comment:
Doesn't look like you actually need `messageObj` here; you're just shallow cloning `messages`, so you could probably just do `const json = JSON.stringify(messages, null, 2)` instead.